src/Tizen.Applications.Common/Tizen.Applications/CoreApplication.cs

```diff
@@ -381,6 +382,19 @@ public static async Task<T> Post<T>(Func<T> runner)
             return await task.Task.ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Set a unique language for system-independent applications
+        /// </summary>
+        /// <remarks>
+        /// If a device does not support the language , It will be set to the default language(en_US.UTF-8)
+        /// </remarks>
+        /// <param name="cultureInfo">The locale to set language</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static void SetApplicationLanguage(CultureInfo cultureInfo)
+        {
+            LocaleManager.SetApplicationLocale(cultureInfo);
+        }
```

Comment:
Please remove this implementation. The developer should use LocaleManager.SetApplicationLanguage().

Reply:
Done. Thank you :-)